src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -1102,6 +1102,7 @@ def load_arguments(self, _):
         c.argument('sku', arg_type=static_web_app_sku_arg_type)
     with self.argument_context('staticwebapp functions link') as c:
         c.argument('function_resource_id', help="Resource ID of the functionapp to link. Can be retrieved with 'az functionapp --query id'")
+        c.argument('environment_name', options_list=['--environment-name'], help="Name of the environment of static site")
```

Comment:
It is unnecessary to repeatedly define the default option name
```suggestion
        c.argument('environment_name', help="Name of the environment of static site")
```